### DISCUSSION TITLE
Add option for satellites to auto-reproduce like units

### DISCUSSION THREAD

**Discussion by JaggedOne** [induktio/thinker#62] [Ideas]
When you create a unit (infantry, vehicle, etc), if you don't make another entry in the queue behind it, it will continue to manufacture said unit infinitely. When I played the game at some time in the past, satellites worked the same way. Now, with the GOG release, satellites no longer do this; thus, I have to enter a new one every time one completes. It would be great if Thinker could add a configuration option to  enable the old-fashioned "auto replicate" behavior for satellites.

**Comment by induktio** [comment]
It could be changed but not sure if there's significant reason to do so. Satellites get treated the same as other facilities in this regard meaning they can be built only once before the queue switches to Stockpile Energy. Nothing prevents the player from filling the queue with satellites though.

**Comment by JaggedOne** [comment]
Yes, I know, I just got used to it repeating on its own back in the day,
and liked it that way.
By the time I am building satellites, I am usually done building everything
else.

Thanks for your reply !!!

<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.www.avg.com
<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>
